src/database/seeders/20201118073216-demo-user.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('Users', [
+      {
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'john@gmail.com',
+        nationalId: 1199480012023222,
+        password: '123abc',
+        phone: '0788767562',
+        language: 'en',
+        role: 'admin',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        firstName: 'Guevara',
+        lastName: 'Che',
+        email: 'che@gmail.com',
+        nationalId: 1200280012023224,
+        password: '123abc',
+        role: 'driver',
+        phone: '0788767561',
+        language: 'en',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        firstName: 'Egide',
+        lastName: 'Isabwe',
+        email: 'isabwe@gmail.com',
```

Comment:
Also for best practice, put fake info in your seeders. This can make the owner of this email a victim of email fishing or other cyber attacks.

Reply:
I din't think about that, thank you this is a very useful tip.